examples/README.md

```diff
@@ -11,16 +11,27 @@ Machine Learning Challenge Winning Solutions
 **LightGBM is used in many winning solutions, but this table is updated very infrequently.**
 
 | Place         | Competition   | Solution  | Date |
-| ------------- |:------------- | --------- | -----|
-|1st      | [M5 Forecasting - Uncertainty](https://www.kaggle.com/c/m5-forecasting-uncertainty) | [link](https://www.kaggle.com/c/m5-forecasting-uncertainty/discussion/163368) | 2020.7 |
-|3rd      | [M5 Forecasting - Uncertainty](https://www.kaggle.com/c/m5-forecasting-uncertainty) | [link](https://www.kaggle.com/c/m5-forecasting-uncertainty/discussion/166875) | 2020.7 |
-|3rd      | [ALASKA2 Image Steganalysis](https://www.kaggle.com/c/alaska2-image-steganalysis) | [link](https://www.kaggle.com/c/alaska2-image-steganalysis/discussion/168870) | 2020.7 |
-|1st      | [M5 Forecasting - Accuracy](https://www.kaggle.com/c/m5-forecasting-accuracy) | [link](https://www.kaggle.com/c/m5-forecasting-accuracy/discussion/163684) | 2020.6 |
-|2nd      | [COVID19 Global Forecasting (Week 5)](https://www.kaggle.com/c/covid19-global-forecasting-week-5) | [link](https://www.kaggle.com/c/covid19-global-forecasting-week-5/discussion/143893) | 2020.5 |
-|3rd      | [COVID19 Global Forecasting (Week 5)](https://www.kaggle.com/c/covid19-global-forecasting-week-5) | [link](https://www.kaggle.com/c/covid19-global-forecasting-week-5/discussion/143029) | 2020.5 |
-|1st      | [COVID19 Global Forecasting (Week 4)](https://www.kaggle.com/c/covid19-global-forecasting-week-4) | [link](https://www.kaggle.com/c/covid19-global-forecasting-week-5/discussion/154804) | 2020.5 |
-|2nd      | [COVID19 Global Forecasting (Week 4)](https://www.kaggle.com/c/covid19-global-forecasting-week-4) | [link](https://www.kaggle.com/c/covid19-global-forecasting-week-5/discussion/144081) | 2020.5 |
-|2nd      | [2019 Data Science Bowl](https://www.kaggle.com/c/data-science-bowl-2019) | [link](https://www.kaggle.com/c/data-science-bowl-2019/discussion/127388) | 2020.1 |
+|---------|:------------- | --------- | -----|
+| 1st     | [Water Supply Forecast Rodeo: Forecast Stage](https://drivendata.co/blog/water-supply-forecast-and-final-winners) |  [link](https://github.com/drivendataorg/water-supply-forecast-rodeo/blob/main/forecast/1st%20place/) | 2024.3 |
```

Comment:
Thanks for your interest in LightGBM.

I noticed this first one does not appear to use LightGBM at all, only CatBoost:

https://github.com/drivendataorg/water-supply-forecast-rodeo/blob/7f152a028a764cdc06b18a14568fabde298ade3e/forecast/1st%20place/training/train_yearly_model.py#L3

I think you've misunderstood the goal of this table... it's not a generic record of data science competitions, but specifically a list of competitions where LightGBM was used in high-ranking submissions.

So before we review any more of these... did all the other entries you added here use LightGBM? If not please remove any that didn't and we'd be happy to revie again.